global.json

```diff
@@ -0,0 +1,7 @@
+{
+    "sdk": {
```

Comment:
I generally don't pin my SDK versions, but I'm not opposed to it either if there's a reason to. Are we using something in this project that benefits from pinning and might change with later SDKs, or just adding this as a best practice?

Reply:
So, it's to ensure more predictability with local builds. On CI you can control which version is exactly used, not on local builds. But from past experience, I have found some users and also myself had installed a recent .NET preview that could break entirely the build of a project. So usually, I don't like to chase down this kind of issues 😅

Reply:
Figured that was it, but wanted to be sure I wasn't missing some crazy version compatibility thing that requires a specific SDK to build. Glad it's just plain one .NET build tooling touchiness!